### ISSUE TITLE
`en-GB` translations for preset names

### ISSUE THREAD

**Issue by GeorgeHoneywood** [Zverik/every_door#353]
My phone is set to the English (United Kingdom)/en-GB locale. When adding a new amenity by searching through the preset list, for example with the input “toilets”, the first result is “Restroom”. This is the correct preset, but “Restroom” is the “en” translation of the `amenity/toilets` preset, not the “en-GB” one. Most people in the UK will understand the US terms, but it is a little confusing.

This behaviour seems to come from [here in `presets.dart`](https://github.com/Zverik/every_door/blob/main/lib/providers/presets.dart#L94). For some reason `locale.toLanguageTag()` is returning `en` instead of `en-GB`. If I manually add a line like `langs.add("en-GB");`, I get the “en-GB” translated results.

If I add an empty `app_en_GB.arb` file (with contents of `{}`), it seems to make `locale.toLanguageTag()` return `en-GB` like it should, and I get the translated preset results. Not sure if this is a particularly good fix though.

**Comment by Zverik** [comment]
Hey George, you are right, `en-GB` locale is not currently supported. For it to be, you need to go to [Weblate](https://hosted.weblate.org/projects/every-door/app/) and add it specifically. Also please check that words indeed use British spelling. After that `en-GB` should be correctly detected.

**Comment by GeorgeHoneywood** [comment]
Thanks Ilya, I'll have a go at that :)

**Comment by GeorgeHoneywood** [comment]
@Zverik It seems Weblate only lets you add new "basic" languages, not language variants.

It says "Please contact project maintainers to add others." at the bottom.

Thanks for looking into this!

**Comment by Zverik** [comment]
Done! https://hosted.weblate.org/projects/every-door/app/en_GB/

**Comment by GeorgeHoneywood** [comment]
I've made a couple of translations ("License" to "Licence" and "apartments" to "flats"), but most of it is all good already. Is it better to clone the already correct translations, or just leave them red?

Most of the benefit is going to be in the preset translations — it always seems to confuse me when I search "bin" and the result is "waste basket".

I've not used Weblate (or translated anything before) so apologies if I've made any mistakes!
